aiohttp/file_sender.py

```diff
@@ -152,16 +153,48 @@ def send(self, request, filepath):
         if not ct:
             ct = 'application/octet-stream'
 
-        resp = self._response_factory()
+        file_size = st.st_size
+
+        status = 200
+        start = 0
+        end = file_size - 1
+
+        # Handle 206 range response if requested
+        if 'range' in request.headers:
```

Comment:
I had thought that but wasn't sure where would make sense to put it. 
It could be in a parse_range_header(request) function in helpers, it's not necessarily making things clearer but would certainly would aid in reuse.